resources/ssm_instances.go

```diff
@@ -0,0 +1,267 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func SsmInstances() *schema.Table {
+	return &schema.Table{
```

Comment:
would remove "Member is required." from descriptions

Reply:
Done